lib/person.rb

```diff
@@ -3,12 +3,13 @@
 # Person class represents a trackable person in a city
 # It contains
 class Person
-  attr_accessor :name, :surname, :status
+  attr_accessor :name, :surname, :status, :location
 
-  def initialize(name = 'John', surname = 'Doe')
+  def initialize(name, surname, location)
```

Comment:
Why does a person have a location if it's not the same as his phone location (unless we are implying that person also has a tracking chip inside of him)

Reply:
Aren't we implying that everything has a chip inside of them?